core/src/main/java/org/kiwix/kiwixmobile/core/error/ErrorActivity.kt

```diff
@@ -84,16 +84,18 @@ open class ErrorActivity : BaseActivity() {
   }
 
   private fun emailIntent(): Intent {
+    val fileContent = buildBody()
```

Comment:
hmmm `fileContent`...technically true but only conditionally right? What it actually is is the body of the email and that is always the case

Reply:
It isn't just the email-body but it contains more device info in comparison to that of _body material_, that's why I left it as to be the fileContent, it should've been if it contained only starting note stuff(like hi, hello etc.), I think . Views?

Reply:
`putExtra(Intent.EXTRA_TEXT, fileContent)`

it is the body of the email, it just so happens that the body of the email contains info about the device which I find immaterial to the fact of it being the email contents.

`buildBody()` starts with `initialBody` aka 

```
  protected open val initialBody: String
    get() = """
      Hi Kiwix Developers!
      The Android app crashed, here are some details to help fix it:
      """.trimIndent()
```

I think what you are suggesting is a method `diagnosticDetails` that would encompass only these sections?
```
  ${if (allowCrash.isChecked && exception != null) exceptionDetails() else ""}
  ${if (allowZims.isChecked) zimFiles() else ""}
  ${if (allowLanguage.isChecked) languageLocale() else ""}
  ${if (allowDeviceDetails.isChecked) deviceDetails() else ""}
  ${if (allowFileSystemDetails.isChecked) systemDetails() else ""}
```

Regardless I am the consumer of this and I don't care about the log file having a tiny pre-amble. renaming to `emailBody` is sufficient

Reply:
Okay fine, I'm convinced about it and understood too because of the exhaustive dissection provided.